plugins/woocommerce/tests/api-core-tests/tests/data/data-crud.test.js

```diff
@@ -7787,112 +7787,56 @@ test.describe( 'Data API tests', () => {
-						name: 'Venezuelan bol&iacute;var (2008–2018)',
-						symbol: 'Bs F',
-						_links: {
-							self: [
-								{
-									href: expect.stringContaining(
-										'data/currencies/VEF'
-									),
-								},
-							],
-							collection: [
-								{
-									href: expect.stringContaining(
-										'data/currencies'
-									),
-								},
-							],
-						},
-					} ),
-				] )
-			);
-		} else {
-			expect( responseJSON ).toEqual(
-				expect.arrayContaining( [
-					expect.objectContaining( {
-						code: 'VEF',
-						name: 'Venezuelan bol&iacute;var',
-						symbol: 'Bs F',
-						_links: {
-							self: [
-								{
-									href: expect.stringContaining(
-										'data/currencies/VEF'
-									),
-								},
-							],
-							collection: [
-								{
-									href: expect.stringContaining(
-										'data/currencies'
-									),
-								},
-							],
-						},
-					} ),
-				] )
-			);
-		}
-		if ( ! shouldSkip ) {
-			expect( responseJSON ).toEqual(
-				expect.arrayContaining( [
-					expect.objectContaining( {
-						code: 'VES',
-						name: 'Venezuelan bol&iacute;var',
-						symbol: 'Bs.',
-						_links: {
-							self: [
-								{
-									href: expect.stringContaining(
-										'data/currencies/VES'
-									),
-								},
-							],
-							collection: [
-								{
-									href: expect.stringContaining(
-										'data/currencies'
-									),
-								},
-							],
-						},
-					} ),
-				] )
-			);
-		} else {
-			expect( responseJSON ).toEqual(
-				expect.arrayContaining( [
-					expect.objectContaining( {
-						code: 'VES',
-						name: 'Bol&iacute;var soberano',
-						symbol: 'Bs.S',
-						_links: {
-							self: [
-								{
-									href: expect.stringContaining(
-										'data/currencies/VES'
-									),
-								},
-							],
-							collection: [
-								{
-									href: expect.stringContaining(
-										'data/currencies'
-									),
-								},
-							],
-						},
-					} ),
-				] )
-			);
-		}
+		expect( responseJSON ).toEqual(
+			expect.arrayContaining( [
+				expect.objectContaining( {
+					code: 'VEF',
+					name: 'Venezuelan bol&iacute;var (2008–2018)',
+					symbol: 'Bs F',
+					_links: {
+						self: [
+							{
+								href: expect.stringContaining(
+									'data/currencies/VEF'
+								),
+							},
+						],
+						collection: [
+							{
+								href: expect.stringContaining(
+									'data/currencies'
+								),
+							},
+						],
+					},
+				} ),
+			] )
+		);
+		expect( responseJSON ).toEqual(
+			expect.arrayContaining( [
+				expect.objectContaining( {
+					code: 'VES',
+					name: 'Venezuelan bol&iacute;var',
+					symbol: 'Bs.',
+					_links: {
+						self: [
+							{
+								href: expect.stringContaining(
+									'data/currencies/VES'
+								),
+							},
+						],
+						collection: [
+							{
+								href: expect.stringContaining(
+									'data/currencies'
+								),
+							},
+						],
+					},
+				} ),
+			] )
+		);
```

Comment:
This was a conditional I had in before because of a currency change that was pending.  That's been merged now, so updated this test.